src-java/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/feature/HalfSizeMetadataFeature.java

```diff
@@ -0,0 +1,37 @@
+/* Copyright 2020 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.floodlight.feature;
+
+import org.openkilda.model.SwitchFeature;
+
+import net.floodlightcontroller.core.IOFSwitch;
+
+import java.util.Optional;
+
+public class HalfSizeMetadataFeature extends NoviflowSpecificFeature {
+    @Override
+    public Optional<SwitchFeature> discover(IOFSwitch sw) {
+        boolean match = E_SWITCH_MANUFACTURER_DESCRIPTION.equalsIgnoreCase(getManufacturer(sw));
```

Comment:
why is it still E_SWITCH_MANUFACTURER_DESCRIPTION match first? isNoviSwitch by itself should be enough

Reply:
Correct. I forgot to change this feature-detector.